Update Readme.md (#1)
DevExpress-Examples/aspxpivotgrid-group-date-time-values: Readme.md

<!-- default badges list -->
[![](https://img.shields.io/badge/Open_in_DevExpress_Support_Center-FF7200?style=flat-square&logo=DevExpress&logoColor=white)](https://supportcenter.devexpress.com/ticket/details/E1875)
[![](https://img.shields.io/badge/📖_How_to_use_DevExpress_Examples-e9f6fc?style=flat-square)](https://docs.devexpress.com/GeneralInformation/403183)
<!-- default badges end -->
# Pivot Grid for Web Forms - How to Group Date-Time Values

This example shows how to group date-time values.

![Pivot Grif for Web Forms - Group Date-Time Values](images/pivot-grid-web-forms-group-date-time.png)

> In the example, the [ASPxPivotGrid](https://docs.devexpress.com/AspNet/DevExpress.Web.ASPxPivotGrid.ASPxPivotGrid) contains two [PivotGridField](https://docs.devexpress.com/AspNet/DevExpress.Web.ASPxPivotGrid.PivotGridField) fields (*fieldYear* and *fieldMonth*) located in the Column Header Area. Each of the field is bound to the "OrderDate" data field. The first pivot grid field's values are grouped by years, the second field's values are grouped by months. To specify the grouping mode, use the binding’s [DataSourceColumnBindingBase.GroupInterval](https://docs.devexpress.com/CoreLibraries/DevExpress.PivotGrid.DataBinding.DataSourceColumnBindingBase.GroupInterval) property. 

Reviewer: ```suggestion
In the example, the [ASPxPivotGrid](https://docs.devexpress.com/AspNet/DevExpress.Web.ASPxPivotGrid.ASPxPivotGrid) contains two [PivotGridField](https://docs.devexpress.com/AspNet/DevExpress.Web.ASPxPivotGrid.PivotGridField) fields (*fieldYear* and *fieldMonth*) located in the Column Header Area. Each field is bound to the "OrderDate" data field. The first pivot grid field's values are grouped by years, and the second field's values by months. To specify the grouping mode, use the binding’s [DataSourceColumnBindingBase.GroupInterval](https://docs.devexpress.com/CoreLibraries/DevExpress.PivotGrid.DataBinding.DataSourceColumnBindingBase.GroupInterval) property. 

```

[![](https://img.shields.io/badge/Open_in_DevExpress_Support_Center-FF7200?style=flat-square&logo=DevExpress&logoColor=white)](https://supportcenter.devexpress.com/ticket/details/E1875)
[![](https://img.shields.io/badge/📖_How_to_use_DevExpress_Examples-e9f6fc?style=flat-square)](https://docs.devexpress.com/GeneralInformation/403183)
<!-- default badges end -->
# Pivot Grid for Web Forms - How to Group Date-Time Values

This example shows how to group date-time values.

> In the example, the [ASPxPivotGrid](https://docs.devexpress.com/AspNet/DevExpress.Web.ASPxPivotGrid.ASPxPivotGrid) control is bound to a data table that contains an 'OrderDate' field. Two PivotGridField fields are created and displayed in the Column Header Area, each of them bound to the 'OrderDate' field. The first pivot grid field's values are grouped by years, the second field's values are grouped by months. To group data, the [PivotGridFieldBase.GroupInterval](https://docs.devexpress.com/CoreLibraries/DevExpress.XtraPivotGrid.PivotGridFieldBase.GroupInterval) property is used.

Reviewer 2: ```suggestion
In the example, the [ASPxPivotGrid](https://docs.devexpress.com/AspNet/DevExpress.Web.ASPxPivotGrid.ASPxPivotGrid) contains two PivotGridField fields located in the Column Header Area. Each of the field is bound to the "OrderDate" data field. The first pivot grid field's values are grouped by years, the second field's values are grouped by months. To group data, specify the [PivotGridFieldBase.GroupInterval](https://docs.devexpress.com/CoreLibraries/DevExpress.XtraPivotGrid.PivotGridFieldBase.GroupInterval) property.

```
